LifeCycleAssessment_Engine/Compute/EvaluateEnvironmentalProductDeclaration.cs

```diff
@@ -284,84 +105,90 @@ public static double EvaluateLifeCycleAssessment(ProjectLifeCycleAssessment lca,
         [Input("environmentalProductDeclaration", "This is LifeCycleAssessment.EnvironmentalProductDeclaration data. Please select your desired dataset and supply your material choice to the corresponding BHoM objects.")]
         [Input("environmentalProductDeclarationField", "Filter the provided EnvironmentalProductDeclaration by selecting one of the provided metrics for calculation. This method also accepts multiple fields simultaneously.")]
         [Output("quantity", "The quantity of the desired metric provided by the EnvironmentalProductDeclarationField")]
-        public static double EvaluateEnvironmentalProductDeclarationPerObject(IBHoMObject obj, IEnvironmentalProductDeclarationData environmentalProductDeclaration, EnvironmentalProductDeclarationField environmentalProductDeclarationField = EnvironmentalProductDeclarationField.GlobalWarmingPotential) //default to globalWarmingPotential evaluation
+        public static double EvaluateEnvironmentalProductDeclarationPerObject(IBHoMObject obj, EnvironmentalProductDeclarationField field = EnvironmentalProductDeclarationField.GlobalWarmingPotential)
```

Comment:
I'll be honest, I'm not a fan of this sort of method where we attempt to get the volume (or other property) from as many sources as possible. The purpose of `IElementM` is to reduce the need for this, so I would prefer to see this sort of method rely on the `IElementM` interface and associated methods only. There is a danger of people putting in bad data by mistake and getting bad assessments out